.gitignore

```diff
@@ -42,6 +42,7 @@ wheels/
 .installed.cfg
 *.egg
 MANIFEST
+packages-microsoft-prod.deb
```

Comment:
what's that about?

Reply:
To run end-to-end testing I had to install the dotnet framework on my WSL linux. Having done that and run the test this ended up in the top level directory (I guess it was downloaded there as part of the installation). I can delete it but just to be safe that the same does not happen to someone else too, I added it to the .gitignore.